assets/js/nebula.js

```diff
@@ -2728,23 +2728,34 @@ nebula.cf7LocalStorage = function(){
 		return false;
 	}
 
-	jQuery('.wpcf7-textarea, .wpcf7-text').each(function(){
+	jQuery('.wpcf7-textarea, .wpcf7-text, .wpcf7-select, .wpcf7-checkbox input').each(function(){
 		var thisLocalStorageVal = localStorage.getItem('cf7_' + jQuery(this).attr('name'));
 
-		//Fill textareas with localstorage data on load
+		//Complete inputs with localstorage data on load
 		if ( !jQuery(this).hasClass('do-not-store') && !jQuery(this).hasClass('.wpcf7-captchar') && thisLocalStorageVal && thisLocalStorageVal !== 'undefined' && thisLocalStorageVal !== '' ){
-			if ( jQuery(this).val() === '' ){ //Don't overwrite a field that already has text in it!
+			if( jQuery(this).attr('type') == 'checkbox' ){
+				// local storange vals are stored as strings but the .prop method requires a bool
+				if( thisLocalStorageVal == 'true' ){
+					jQuery(this).prop('checked', true);
+				}else{
+					jQuery(this).prop('checked', false);
+				}
+			}else if( jQuery(this).val() === '' ){ //Don't overwrite a field that already has text in it!
 				jQuery(this).val(thisLocalStorageVal).trigger('keyup');
-			}
-			jQuery(this).blur();
+				jQuery(this).blur();
```

Comment:
Just looking at this in my head I think this `.blur()` may need to be outside of the conditional– that way it blurs every input regardless.

Reply:
Actually I put that inside there while troubleshooting something else and looks like I left it after I fixed whatever. I don't know if it was related but I'll move it out and see if it mattered.

Reply:
This doesn't seem to break anything by itself